docs/input/_Bottom.cshtml

```diff
@@ -1,12 +1,10 @@
 <div class="github-button">
     <a href="https://github.com/cake-contrib/Cake.Npm" target="_blank"><i class="fa fa-github"></i> GitHub</a>
 </div>
-<script>
-    ((window.gitter = {}).chat = {}).options = {
-        room: 'cake-contrib/Lobby'
-    };
-</script>
-<script src="https://sidecar.gitter.im/dist/sidecar.v1.js" async defer></script>
+<a class="discussion-button" href="https://github.com/cake-build/cake/discussions" target="_blank">
```

Comment:
Link should go to the correct category: https://github.com/cake-build/cake/discussions/categories/extension-q-a